cheddar/cheddar-events/build.gradle

```diff
@@ -3,6 +3,7 @@ def springVersion = '4.1.1.RELEASE'
 dependencies {
     compile project(':cheddar:cheddar-messaging')
     compile 'com.fasterxml.jackson.core:jackson-databind:2.2.2'
+    compile 'com.fasterxml.jackson.datatype:jackson-datatype-joda:2.4.0'
```

Comment:
I think it is safe to use 2.6.3 for all Jackson dependencies throughout Cheddar. They all appear to work with our other dependencies, notably AWS Java SDK